cpp/src/io/parquet/experimental/hybrid_scan_helpers.hpp

```diff
@@ -50,53 +50,6 @@ struct metadata : private metadata_base {
 };
 
 class aggregate_reader_metadata : public aggregate_reader_metadata_base {
- private:
```

Comment:
These will likely update when dictionary based row group pruning is implemented so removing them for now.